content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -238,6 +239,15 @@ it adds a label, `statefulset.kubernetes.io/pod-name`, that is set to the name o
 the Pod. This label allows you to attach a Service to a specific Pod in
 the StatefulSet.
 
+### Pod Index Label
+
+{{< feature-state for_k8s_version="v1.28" state="beta" >}}
+
+When the StatefulSet {{<glossary_tooltip text="controller" term_id="controller">}} creates a Pod,
+it adds a label, `apps.kubernetes.io/pod-index`, that is set to the ordinal index of
```

Comment:
Avoid a risk that people infer “it” to be ”the new Pod” here.
```suggestion
the new Pod is labelled with `apps.kubernetes.io/pod-index`. The value of this label is the ordinal index of
```

Reply:
I unresolved this conversation as my concern stands.

Reply:
sorry about that I thought I hit "commit suggestion" already but I guess not. Thanks!